pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -217,8 +242,11 @@ func EvalAddSSTable(
 		desc := cArgs.EvalCtx.Desc()
 		leftPeekBound, rightPeekBound := rangeTombstonePeekBounds(
 			args.Key, args.EndKey, desc.StartKey.AsRawKey(), desc.EndKey.AsRawKey())
-		statsDelta, err = storage.CheckSSTConflicts(ctx, sst, readWriter, start, end, leftPeekBound, rightPeekBound,
+		statsDelta, err = storage.CheckSSTConflicts(ctx, sst, readWriter, start, end, leftPeekBound, rightPeekBound, h.Timestamp.WallTime,
 			args.DisallowShadowing, args.DisallowShadowingBelow, maxIntents, usePrefixSeek)
+		if sstReqStatsUpdated {
```

Comment:
nit: this variable probably isn't necessary, since adding an empty struct should be a noop.